### DISCUSSION TITLE
Is it possible to specify root partition size without using RuntimeSize?

### DISCUSSION THREAD

**Discussion by adrianlshaw** [systemd/mkosi#2898] [Q&A]
My workflow is to make a bootable image using mkosi, and then use raw qemu on the command line.
Mkosi will default to making a small image, and as far I can see from the documentation, to make it bigger I need to use RuntimeSize with mkosi boot. Sadly this breaks my setup.

**Comment by daandemeyer** [comment]
@adrianlshaw You can override the default partitions and set the partition sizes explicitly. For example, in systemd we have a custom `mkosi.repart/` directory with the following files:

00-esp.conf:

```
[Partition]
Type=esp
Format=vfat
CopyFiles=/boot:/
CopyFiles=/efi:/
SizeMinBytes=1G
SizeMaxBytes=1G
```

10-root.conf:

```
[Partition]
Type=root
CopyFiles=/
SizeMinBytes=8G
SizeMaxBytes=8G
```

You can adjust the sizes to fit your needs.

**Reply by adrianlshaw** [reply]
Hi Daan thanks for this! Would this still work when `Bootable=yes` is in mkosi.default?

**Reply by daandemeyer** [reply]
@adrianlshaw Yes

**Reply by daandemeyer** [reply]
@adrianlshaw Note that it's `mkosi.conf` now instead of `mkosi.default`.
